### DISCUSSION TITLE
Is there any way to achieve facial recognition function?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#3525] [Q&A] [answered]
I remember there was a library that could be implemented, but I can't find it now, who knows?
thank you.

**Comment by FischMich** [comment, marked as answer]
I remember this one:

Oskar Wicha's ActionScript 3 face detection and face recognition library using Eigenfaces

https://code.google.com/archive/p/face-recognition-library-as3/

**Reply by Jason0247** [reply]
thanks
